test/Project.toml

```diff
@@ -7,6 +7,7 @@ DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
@Krastanov I saw this was removed in #126 but I don't have the full context for the problem it was solving.

Alternatively, re-adding `using Pkg; Pkg.add("GLMakie")` lines to the plotting test `@testitems` should work as well I think, and is closer to what was already there.

Also, when running tests from VSCode, these tests pass, however when manually running them via `runtests.jl` it was complaining:
```
LoadError: Invalid attribute markerspace for plot type 
MakieCore.LineSegments{Tuple{Vector{GeometryBasics.Point{2, Float32}}}}.
```
have you seen that before ?